sdk_container/src/third_party/coreos-overlay/sys-apps/ignition/files/0001-sed-s-coreos-flatcar.patch

```diff
@@ -426,34 +425,12 @@ Signed-off-by: Mathieu Tortuyaux <mtortuyaux@microsoft.com>
  tests/positive/systemd/modify_service.go      |  4 +-
  tests/positive/timeouts/timeouts.go           |  4 +-
  tests/register/register.go                    | 16 +++---
- tests/registry/registry.go                    | 40 +++++++-------
+ tests/registry/registry.go                    | 40 ++++++-------
  tests/servers/servers.go                      |  2 +-
  tests/validator.go                            |  4 +-
  validate/main.go                              |  4 +-
- 425 files changed, 1130 insertions(+), 1130 deletions(-)
+ 424 files changed, 1133 insertions(+), 1133 deletions(-)
 
-diff --git a/build b/build
-index 9bf93d61..5abdcf5f 100755
---- a/build
-+++ b/build
-@@ -5,7 +5,7 @@ set -eu
- export GO111MODULE=on
- 
- NAME="ignition"
--ORG_PATH="github.com/coreos"
-+ORG_PATH="github.com/flatcar"
- REPO_PATH="${ORG_PATH}/${NAME}/v2"
- GLDFLAGS=${GLDFLAGS:-}
- export GOFLAGS=-mod=vendor
-@@ -15,7 +15,7 @@ if [ -z ${VERSION+a} ]; then
- 	echo "Using version from git: $VERSION"
- fi
- 
--GLDFLAGS+="-X github.com/coreos/ignition/v2/internal/version.Raw=${VERSION}"
-+GLDFLAGS+="-X github.com/flatcar/ignition/v2/internal/version.Raw=${VERSION}"
- 
- eval $(go env)
- 
```

Comment:
I suppose that this part is dropped from the patch because we don't use the build script?

Reply:
> Yeah, it was a too large `sed` before. It's now scoped to `*.go` files only.

Sounds good. But if you have a script that automates it, then possibly add `go.mod` to it too then.

> And correct, we use the `go_build` command from the `coreos-go` (which could actually be dropped in favor of regular Go eclass)

Right, eventually. ;)